Project.toml

```diff
@@ -14,6 +14,7 @@ DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
 FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
 FiniteDiff = "6a86dc24-6348-571c-b903-95158fe2bd41"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
+LevyArea = "2d8b4e74-eb68-11e8-0fb9-d5eb67b50637"
```

Comment:
This needs a compat bound, otherwise you won't be able to add a new release to the registry.